python/python/glide/async_commands/server_modules/json.py

```diff
@@ -84,6 +84,33 @@ def to_args(self) -> List[str]:
         return args
 
 
+class JsonArrPopOptions:
+    """
+    Options for the JSON.ARRPOP command.
+
+    Args:
+        path (TEncodable): The path within the JSON document.
+        index (Optional[int]): The index of the element to pop. If not specified, will pop the last element.
+            Out of boundary indexes are rounded to their respective array boundaries. Defaults to None.
```

Comment:
It seems as the partial copy from the redis docs, what about the poping from the empy array?

Reply:
see line 45s, popes None